llvm-bazel/llvm-project-overlay/mlir/BUILD

```diff
@@ -163,7 +163,8 @@ cc_library(
         "include/mlir-c/Bindings/Python/Interop.h",
     ],
     tags = [
-        "manual",  # TODO(gcmn): Add support for this target
+        "manual",  # External dependency
+        "nobuildkite",  # TODO(gcmn): Add support for this target
```

Comment:
Should maybe have a canonical order and comment structure? (Differs from the above, not suggesting which one is best)

Reply:
Hmm I'm surprised buildifier isn't sorting them. These are different though. In this case the manual tag is there because this target uses an external dependency, so we want it to only get built when requested (to avoid pulling in that external dependency) even when we do get it to build. In some cases, the manual tag is only there because the target is currently broken, but once we get it working we should be able to remove it entirely.

Reply:
Cleaned up the comments.